.github/workflows/ci.yml

```diff
@@ -78,16 +44,18 @@ jobs:
         run: |
           set -ex
           ./scripts/setup-ci.sh
+          ./scripts/setup-dev.sh
 
-      - name: "Build Debug with clang-tidy"
+      - name: "Run Checks and Build Debug with clang-tidy"
         run: |
           set -ex
           git config --global --add safe.directory /__w/CCF/CCF
+          ./scripts/ci-checks.sh
           mkdir build
           cd build
           cmake -GNinja -DCMAKE_BUILD_TYPE=Debug -DCLANG_TIDY=ON ..
           ninja
-        shell: bash
+          shell: bash
```

Comment:
```suggestion
        shell: bash
```
Don't know how this indentation appeared, but it's resulting in bad syntax and a broken run
<img width="681" height="98" alt="image" src="https://github.com/user-attachments/assets/b30ec299-1c0e-4f54-8231-4f8db12eaf05" />